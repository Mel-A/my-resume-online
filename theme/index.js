const theme = {
  colors: {
    alabaster: '#FAFAFA',
    astral: '#327DA0',
    aquamarine: '#A1DAD7',
    azure: '#0080FF',
    cobalt: '#003DB3',
    gray: '#333333',
    white: '#FFFFFF'
  },
  fonts: {
    small0: '10px',
    small8: '18px',
    medium0: '20px',
    medium4: '24px',
    big0: '30px'
  }
}

export default theme