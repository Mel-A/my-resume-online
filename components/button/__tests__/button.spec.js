import { shallow, mount } from 'enzyme'
import Button from '..'
import theme from '../../../theme'
import 'jest-styled-components'
import { ThemeProvider } from "styled-components"
import renderer from 'react-test-renderer'

describe('Button', () => {
  it('button renders correctly', () => {
    const button = shallow(<Button theme={theme} />)
    expect(button).toMatchSnapshot()
  })
})
