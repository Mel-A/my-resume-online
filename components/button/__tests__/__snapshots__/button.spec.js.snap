// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button button renders correctly 1`] = `ShallowWrapper {}`;
